lib/client.js

```diff
@@ -70,7 +70,8 @@ const api = ({
       const bodyText = await response.text();
       let body = bodyText;
 
-      switch (response.headers.get('content-type')) {
+      const [contentType] = (response.headers.get('content-type') || '').split(';');
```

Comment:
is `contentType` not an array? How are you switching over an array?

Reply:
No but but the header can contain multiple parts:

`Content-Type: application/json` `Content-Type: application/json;charset=utf-8`

This helps make sure an backend update is not breaking the decoding